llm_bench/python/benchmark.py

```diff
@@ -453,6 +468,12 @@ def get_argprser():
     parser.add_argument(
         '--convert_tokenizer', action='store_true', help='Convert tokenizer to OpenVINO format'
     )
+    parser.add_argument(
```

Comment:
Shall we add a default=False?

Reply:
We add action='store_true' means if ‘--interleave‘ is given on the command line, its value is True, otherwise it is False. 
Reference --convert_tokenizer.